### PR TITLE
Book Model

### PR DESCRIPTION
**Pull request by nicoopalenque**
# Sumary

* Book's model implement
* Setting fields to required
* create books with `rails c`

# Card

[https://trello.com/c/kBMyjdf3](url)

# Screenshot

Captura de Pantalla 2021-06-04 a la(s) 09 28 38

### CODE REVIEW

**Review by blacksam07 — APPROVED** [review]

File: spec/models/book_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Book, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (redwarewolf) [inline spec/models/book_spec.rb:4]: You can add the [shoulda matchers](https://github.com/thoughtbot/shoulda-matchers)
They have tests for validating a model's field presence

**Comment by nicoopalenque** [comment]
I added the `open_library.rb` file in the model directory because if I don't add this will broke in Wor::paginate when I go to the route

File: app/models/open_library.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class OpenLibrary < ApplicationRecord
+end
```
Comment (redwarewolf) [inline app/models/open_library.rb:2]: What's the use for this?

**Review by redwarewolf — CHANGES_REQUESTED** [review]

File: spec/models/book_spec.rb (hunk 1 above)
Comment (nicoopalenque) [reply spec/models/book_spec.rb:4]: this is added in the other PR [https://github.com/wx-inc-trainings/np-rails/pull/9](url)

File: app/models/open_library.rb (hunk 2 above)
Comment (nicoopalenque) [reply app/models/open_library.rb:2]: if I don't add this worpaginate it will break

**Pull request merged**